po/Makefile

```diff
@@ -27,6 +27,8 @@ xgettext -f PYPOTFILES -L Python --force-po --from-code=utf-8 -o messages.pot \
     --package-name=$(APPNAME) --copyright-holder=$(ORG)
 xgettext -f CPOTFILES -L C -k_ -kN_ -j -o messages.pot \
     --package-name=$(APPNAME) --copyright-holder=$(ORG)
+xgettext -o messages.pot -j -L C --keyword=Q_ --keyword=N_ ../lxpanel-plugin/*.c  \
```

Comment:
Yeah, the Gtk Internationalisation page [recommends it over `_()`](https://developer.gnome.org/glib/stable/glib-I18N.html#Q-:CAPS):

> This has the advantage that the string can be adorned with a prefix to guarantee uniqueness and provide context to the translator.

Don't really know what it means but if I did then it wouldn't be Gtk!